python/ccao/vars_funs.py

```diff
@@ -0,0 +1,156 @@
+# Functions for translating variables between different data sources
+import enum
+import importlib.resources
+import typing
+
+import pandas as pd
+
+import ccao.data
+
+# Load the default variable dictionary
+_data_path = importlib.resources.files(ccao.data)
+vars_dict = pd.read_csv(str(_data_path / "vars_dict.csv"))
+
+# Prefix we use to identify variable name columns in the variable dictionary
+var_name_prefix = "var_name"
+
+
+class OutputType(enum.Enum):
+    """Possible output types for variable renaming"""
+
+    INPLACE = "inplace"
+    VECTOR = "vector"
+
+    @classmethod
+    def values(cls) -> list[str]:
+        """Get the possible values for this enum"""
+        return [type_.value for type_ in cls]
+
+
+def vars_rename(
```

Comment:
Compare to: https://github.com/ccao-data/ccao/blob/d85e0d4729b12f1bd2bc200b0b8a93ccca5cdd38/R/vars_funs.R#L137